test/Style.js

```diff
@@ -292,70 +298,89 @@ test.serial('if Style will render correctly with default props', (t) => {
+  (t) => {
+    const props = {
+      children: '.foo { display: flex; }',
+      'data-foo': 'bar'
+    };
 
-  const wrapper = shallow(<Style {...props} />);
+    const wrapper = shallow(<Style {...props} />);
 
-  t.snapshot(toJson(wrapper));
+    t.snapshot(toJson(wrapper));
 
-  blob.getUrl.reset();
-  blob.hasBlobSupport.reset();
-});
+    blob.getUrl.reset();
+    blob.hasBlobSupport.reset();
+  },
+);
 
-test.serial('if Style will render correctly with default props when blob support does not exist', (t) => {
-  const props = {
-    children: '.foo { display: flex; }'
-  };
+test.serial(
+  'if Style will render correctly with default props when blob support does not exist',
+  (t) => {
+    const props = {
+      children: '.foo { display: flex; }'
+    };
 
-  const existingBlob = window.Blob;
+    const existingBlob = window.Blob;
 
-  window.Blob = undefined;
+    window.Blob = undefined;
 
-  const stub = sinon.stub(console, 'error');
+    const stub = sinon.stub(console, 'error');
 
-  const wrapper = shallow(<Style {...props} />);
+    const wrapper = shallow(<Style {...props} />);
 
-  t.true(stub.calledOnce);
+    t.true(stub.calledOnce);
 
-  stub.restore();
+    stub.restore();
 
-  t.snapshot(toJson(wrapper));
+    t.snapshot(toJson(wrapper));
 
-  blob.getUrl.reset();
-  blob.hasBlobSupport.reset();
+    blob.getUrl.reset();
+    blob.hasBlobSupport.reset();
 
-  window.blob = existingBlob;
-});
+    window.blob = existingBlob;
+  },
+);
 
-test.serial('if Style will render correctly with no sourceMap requested', (t) => {
-  const props = {
-    children: '.foo { display: flex; }',
-    hasSourceMap: false
-  };
+test.serial(
+  'if Style will render correctly with no sourceMap requested',
+  (t) => {
+    const props = {
+      children: '.foo { display: flex; }',
+      hasSourceMap: false
+    };
 
-  const wrapper = shallow(<Style {...props} />);
+    const wrapper = shallow(<Style {...props} />);
 
-  t.snapshot(toJson(wrapper));
+    t.snapshot(toJson(wrapper));
 
-  blob.getUrl.reset();
-  blob.hasBlobSupport.reset();
-});
+    blob.getUrl.reset();
+    blob.hasBlobSupport.reset();
+  },
+);
 
-test.serial('if Style will render correctly with no sourceMap requested and additional props', (t) => {
-  const props = {
-    children: '.foo { display: flex; }',
-    'data-foo': 'bar',
-    hasSourceMap: false
-  };
+test.serial(
+  'if Style will render correctly with no sourceMap requested and additional props',
+  (t) => {
+    const props = {
+      children: '.foo { display: flex; }',
+      'data-foo': 'bar',
+      hasSourceMap: false
+    };
 
-  const wrapper = shallow(<Style {...props} />);
+    const wrapper = shallow(<Style {...props} />);
 
-  t.snapshot(toJson(wrapper));
+    t.snapshot(toJson(wrapper));
 
-  blob.getUrl.reset();
-  blob.hasBlobSupport.reset();
+    blob.getUrl.reset();
+    blob.hasBlobSupport.reset();
+  },
+);
+
+test.serial('if Style will have the correct will update method', (t) => {
```

Comment:
Added a test to verify that the `UNSAFE_` version is used when the version is late enough.